tests/runtests.py

```diff
@@ -234,6 +235,22 @@ def actual_test_processes(parallel):
         return parallel
 
 
+def validate_selenium(selenium_args):
+    """
+    Validate the browsers requested and raise appropriate exception.
+    """
+    # Remove any browser repition.
+    browsers = list(set(selenium_args.split(',')))
+    for browser in browsers:
+        try:
+            # Create a class instance to see if everything is fine.
+            import_string('selenium.webdriver.%s.webdriver.WebDriver' % browser)().quit()
+        except:
+            msg = 'Selenium specifications not valid or corresponding WebDriver not installed: %s' % browser
```

Comment:
The string `'selenium.webdriver.%s.webdriver.WebDriver' % browser_spec` is being hard-coded at a lot of places now. I think it would be better to make a function in some file which would return this string or maybe the driver itself. A `get_selenium_driver()` in `django/test/selenium.py`? `django/test/utils`?